crates/sdk/src/keygen/mod.rs

```diff
@@ -155,6 +156,77 @@ where
     }
 }
 
+pub fn check_recursive_verifier_size<SC: StarkGenericConfig>(
```

Comment:
Also ideally this is private but I needed it to be public to use in the benchmarks because they don't go through `AggStarkProvingKey::keygen`.